cmd/agent/subcommands/status/command.go

```diff
@@ -209,3 +214,20 @@ func componentStatus(config config.Component, cliParams *cliParams, component st
 
 	return nil
 }
+
+func requestSections(config config.Component) error {
+
+	endpoint, err := apiutil.NewIPCEndpoint(config, "/agent/status/sections")
+	if err != nil {
+		return err
+	}
+
+	res, err := endpoint.DoGet()
+	if err != nil {
+		return err
+	}
+
+	fmt.Println(string(res))
```

Comment:
I think you're printing a marshaled list here ? If so maybe unmarshaling it and printing as text (not json) would be better ?

Reply:
I mean you can just do `for _, section := range sections { fmt.Printf("- \"%s\"\n", section) }`, it mostly depends what the JSON looks like (eg. if it's on a single line I think it would not be very readable, but if it's pretty printed then ok)